build.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
 allprojects {
   repositories {
     jcenter()
+    mavenCentral()
```

Comment:
I added it because [that's what the plugin instructions seemed to say](https://github.com/mapbox/mapbox-plugins-android). Maybe I misread/misunderstood though.